frontend/src/api/api.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+import { getUserToken } from "./auth";
+
+const api = axios.create({
+  baseURL: `${process.env.REACT_APP_IDP_URL}/api`
```

Comment:
It is not REACT_APP_IDP_URL. It should be another REACT_APP_ environment variable for API url

Reply:
what is the API URL then? because locally `REACT_APP_IDP_URL='http://localhost:5001'` is the correct URL.

Reply:
The API URL is http://localhost:5008

Reply:
sounds good, thanks for clarifying !